src/components/Landing.js

```diff
@@ -194,8 +222,9 @@ export default class Landing extends Component {
   processEndPoint(endpoint) {
     if (!endpoint) return "";
     return (endpoint)
-    .replace('{process.env.REACT_APP_CONF_API_URL}', process.env.REACT_APP_CONF_API_URL)
-    .replace('{process.env.PUBLIC_URL}', process.env.PUBLIC_URL);
+    .replace('{process.env.REACT_APP_CONF_API_URL}', getEnv("REACT_APP_CONF_API_URL"))
+    .replace('{process.env.PUBLIC_URL}', getEnv("PUBLIC_URL"))
```

Comment:
Minor nitpick; out of scope of review
I think using string replacement in this way requires keeping this file and `summary.json` in sync.

Assuming that's the case, are you able to evaluate these templates lazily (at run-time)?
[I found this example](https://stackoverflow.com/questions/30003353/can-es6-template-literals-be-substituted-at-runtime-or-reused/37217166#37217166), not sure if relevant

Reply:
Thank you, Ivan, for the great example! I know of the shortcoming with string replacements but wasn’t sure how to fix it, I will try this out when I need to fix this (query HAPI)